src/targets/gpu/convolution.cpp

```diff
@@ -21,9 +21,13 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
+#include <migraphx/env.hpp>
 #include <migraphx/gpu/convolution.hpp>
 #include <migraphx/gpu/context.hpp>
 #include <migraphx/generate.hpp>
+#include <miopen/miopen.h>
+
+MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_MIOPEN_FIND_2)
```

Comment:
We cant use an env variable as this wont build with the older miopen. This should be a cmake flag.

Reply:
Done. But not sure if it would create compatibility issues between MIGraphX  and MIOpen.